dw1000/src/hl/mod.rs

```diff
@@ -12,14 +12,10 @@
 use crate::ll;
 use core::{fmt, num::Wrapping};
 
-pub use awake::*;
 pub use error::*;
 pub use ready::*;
 pub use receiving::*;
-pub use sending::*;
-pub use sleeping::*;
 pub use state_impls::*;
-pub use uninitialized::*;
```

Comment:
Did you intend to remove these?

Reply:
Clippy reported them as unused, and they only contain `impl` blocks which should still be accessible outside without `pub use` as the type itself is `pub`.